src/message/renderMessages.js

```diff
@@ -65,6 +65,7 @@ export default ({ auth, subscriptions, messages, narrow, mute, doNarrow }) => {
         isBrief={shouldGroupWithPrev}
         doNarrow={doNarrow}
         avatarUrl={getFullUrl(item.avatar_url, auth.realm)}
+        handleLongPress={handleLongPress}
```

Comment:
Considering the differences between `on` and `handle` this will be `onLongPress={onLongPress}`